content/zh-cn/docs/reference/kubernetes-api/cluster-resources/event-v1.md

```diff
@@ -0,0 +1,935 @@
+    -->
+    产生事件的节点名称。
+
+- **note** (string)
+
+  <!--
+  note is a human-readable description of the status of this operation. Maximal length of the note is 1kB, but libraries should be prepared to handle values up to 64kB.
+  -->
+  node 是对该操作状态的可读描述。注释的最大长度是 1kB，但是库应该准备好处理最多 64kB 的值。
+
+- **reason** (string)
+
+  <!--
+  reason is why the action was taken. It is human-readable. This field cannot be empty for new Events and it can have at most 128 characters.
+  -->
+  reason 是采取行动的原因。它是人类可读的。对于新的 Event ，此字段不能为空，且最多为128个字符。
+
+- **regarding** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)
+
+  <!--
+  regarding contains the object this Event is about. In most cases it's an Object reporting controller implements, e.g. ReplicaSetController implements ReplicaSets and this event is emitted because it acts on some changes in a ReplicaSet object.
+  -->
+  关于包含此 Event 所涉及的对象。在大多数情况下，所指的是报告事件的控制器所实现的一个 Object。
+  例如，ReplicaSetController 实现了 ReplicaSet，这个事件被触发是因为控制器对 ReplicaSet 对象做了一些变化。
+
+- **related** (<a href="{{< ref "../common-definitions/object-reference#ObjectReference" >}}">ObjectReference</a>)
+
+  <!--
+  related is the optional secondary object for more complex actions. E.g. when regarding object triggers a creation or deletion of related object.
+  -->
+  related 是用于更复杂操作的、可选的、从属性的对象。例如，当 regarding 对象触发 related 对象的创建或删除时。
+
+- **reportingController** (string)
+
+  <!--
+  reportingController is the name of the controller that emitted this Event, e.g. `kubernetes.io/kubelet`. This field cannot be empty for new Events.
+  -->
+  reportingController 是触发该事件的控制器的名称。例如, `kubernetes.io/kubelet`。对于新的　Event，此字段不能为空。
+
+- **reportingInstance** (string)
+
+  <!--
+  reportingInstance is the ID of the controller instance, e.g. `kubelet-xyzf`. This field cannot be empty for new Events and it can have at most 128 characters.
+  -->
+  reportingInstance 为控制器实例的ID。例如，`kubelet-xyzf`。对于新的 Event，此字段不能为空，且最多为 128 个字符。 
+
+- **series** (EventSeries)
+
+  <!--
+  series is data about the Event series this event represents or nil if it's a singleton Event.
+  -->
+  series 是该事件代表的事件系列的数据，如果是单事件，则为 nil。
+
+  <a name="EventSeries"></a>
+
+  <!--
+  *EventSeries contain information on series of events, i.e. thing that was/is happening continuously for some time. How often to update the EventSeries is up to the event reporters. The default event reporter in "k8s.io/client-go/tools/events/event_broadcaster.go" shows how this struct is updated on heartbeats and can guide customized reporter implementations.*
+  -->
+  EventSeries 包含一系列事件的信息，即一段时间内持续发生的事情。
+  EventSeries 的更新频率由事件报告者决定。
+  默认事件报告程序在 "k8s.io/client-go/tools/events/event_broadcaster.go" 
+  展示在发生心跳时该结构如何被更新，可以指导定制的报告者实现。 
+
+  <!--
+  - **series.count** (int32), required
+  -->
+  - **series.count** (int32)，必需
+
+    <!--
+    count is the number of occurrences in this series up to the last heartbeat time.
+    -->
+    Count 是到最后一次心跳时间为止在该系列中出现的次数。
```

Comment:
不好意思，没有理解了什么意思。可不可以给个修改示例，谢谢。

Reply:
Sorry that I did not understand the meaning of Display wrong. Could you please give me an example for modifying it? Thank you.

Reply:
It's parsed as code block instead of list content. I have no idea about how to fix this. 😿
Maybe @tengqm can help with this issue.

我也不知道是哪裡出錯了，但這裡被解析成程式碼區塊而非清單文字，或許 tengqm 知道可以怎麼解決